web/impact/impact/tests/test_graphql.py

```diff
@@ -0,0 +1,62 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from django.urls import reverse
+
+from impact.graphql.middleware import NOT_LOGGED_IN_MSG
+from impact.tests.api_test_case import APITestCase
+from impact.tests.factories import ExpertFactory
+from impact.tests.utils import capture_stderr
+
+
+class TestGraphQL(APITestCase):
+    url = reverse('graphql')
+    auth_url = reverse('graphql-auth')
+
+    def test_anonymous_user_cannot_access_main_graphql_view(self):
+        user = ExpertFactory()
+        query = """query {{ expertProfile(id: {id}) {{ title }} }}
+            """.format(id=user.expertprofile.id)
+
+        with capture_stderr(self.client.post,
+                            self.url,
+                            data={'query': query}) as (response, _):
+            error_messages = [x['message'] for x in response.json()['errors']]
+
+        self.assertIn(NOT_LOGGED_IN_MSG, error_messages)
+
+    def test_anonymous_user_can_access_auth_graphql_view(self):
+        query = """mutation {
+                    verifyToken (token: "some-token") {
+                        payload
+                    }
+                }"""
+        response = self.client.post(self.url, data={'query': query})
+        error_messages = [x['message'] for x in response.json()['errors']]
+        self.assertNotIn(NOT_LOGGED_IN_MSG, error_messages)
+
+    def test_query_with_expert(self):
+        with self.login(email=self.basic_user().email):
+            user = ExpertFactory()
+            query = """
```

Comment:
Not a blocker, but I still don't love the pattern we're setting here. At minimum, I'd like the literal strings to be externalized. Better would be for the query and the expected result to be generated by some function, but that might be too much to ask for, considering the time frame we're in. :)